app/adapters/external/cache/adapter.py

```
@@ -131,10 +134,28 @@ async def ping(self) -> bool:
+            client = self._get_client()
+            keys = client.keys(pattern)
+            # bytes를 문자열로 변환
+            if isinstance(keys, list):
+                return [
+                    key.decode("utf-8") if isinstance(key, bytes) else str(key)
+                    for key in keys
+                ]
+            else:
+                return []
+
+        except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)